# A single-cell 3D spatiotemporal multi-omics atlas from *Drosophila* embryogenesis to metamorphosis
This repo contains custom codes using open-source software for analyses performed in this paper, available as preprint at:  
This work is part of Spatiotemporal Omics Consortium, dedicated to resolving *Drosophila* development in 3D spatiotemporal contexts.  
Currently, our datasets contain:  
1. scStereo-seq data: 3D organism-wide single-cell spatial transcriptomes for 43 embryo, 9 larva, and 5 pupa samples, with a total of 3,812,505 cell bins;
2. scRNA-seq data: 238,242 single-cell transcriptomes, collected at 2h intervals across embryogenesis;
3. scATAC-seq data: 240,573 single-cell chromatin accessibility profiles, collected at 2h intervals across embryogenesis.

## Links to processed matrices
Processed data matrices are available through Mendeley Data.  
Part 1: https://doi.org/10.17632/tvvjfr3c6j.1  
Part 2: https://doi.org/10.17632/29695x8txs.1  
Part 3: https://doi.org/10.17632/4zf847bxcd.1

## Links to databases
These databases contain processed matrices and links to raw sequencing data of this series of study.  
Flysta3D: https://db.cngb.org/stomics/flysta3d/  
Flysta3D V2: *links pending*   

## References
1. Wang, M., Hu, Q., Lv, T., Wang, Y., Lan, Q., Xiang, R., Tu, Z., Wei, Y., Han, K., Shi, C., et al. (2022). High-resolution 3D spatiotemporal transcriptomic maps of developing *Drosophila* embryos and larvae. Dev Cell 57, 1271-1283.e4. 10.1016/j.devcel.2022.04.006.
